### ISSUE TITLE
"--quiet" flag not quiet enough in "forget --prune"

### ISSUE THREAD

**Issue by shane-kerr** [restic/restic#4156]
I have nightly backup runs. They run with `--quiet` and normally produce no output. 3 hosts that got upgraded to 0.15.0 started reporting how many packs are updated, I think during pruning. Nothing was changed on my side, except for upgrading from 0.14.0 to 0.15.0. 

Output of `restic version`
--------------------------
```
restic 0.15.0 compiled with go1.19.5 on linux/arm
```

How did you run restic exactly?
-------------------------------

```
/usr/bin/restic -r sftp::restic-repo -o sftp.command='ssh -l '${USER}' -p '${PORT}' -i /root/.ssh/restic_backup -s '${HOST}' sftp' --password-file=/root/restic_password forget --keep-daily=8 --keep-weekly=9 --keep-monthly=48 --keep-yearly=100 --keep-within 2d --prune --quiet
```

What backend/server/service did you use to store the repository?
----------------------------------------------------------------
`sftp` and `rclone` on Jottacloud. The behavior is identical.

Expected behavior
-----------------
No output.

Actual behavior
---------------
```
[0:12] 100.00%  260 / 260 packs processed
```

Steps to reproduce the behavior
-------------------------------
Run `forget --prune --quiet`. It only seems to produce output if something is pruned, so repeated runs do _not_ show output.

Do you have any idea what may have caused this?
-----------------------------------------------
Not sure. Some refactoring of the code in the 0.15.0 release.

Do you have an idea how to solve the issue?
-------------------------------------------
Not sure.

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------
`restic` makes me happy every day! Especially when it found corrupt data in backups. :heart:

**Comment by mpolden** [comment]
I see the same issue. My weekly `restic --quiet forget` crontab started printing progress output after updating to 0.15.

**Comment by mpolden** [comment]
Looks like this was fixed in #4153.

**Comment by MichaelEischer** [comment]
Duplicate of #4147
